### DISCUSSION TITLE
Modal exit animation doesn't seem to be working in Native Stack

### DISCUSSION THREAD

**Discussion by enlightenedpie** [react-navigation/react-navigation#10393] [Q&A]
Has anyone else come across this scenario? I'm using `createNativeStackNavigator` from `@react-navigation/native-stack`, and any screen that is set to 'presentation: modal' (or any modal type really) does not have an exit animation, the screen just disappears. I've attached a screen cap of what I'm talking about.

This happens whether I use `goBack()`, `replace()`, `pop()`, or even just `navigate()` back to the previous screen.

Has anyone else encountered this, and is there a fix/solve?

https://user-images.githubusercontent.com/8129146/156048064-26a30bb7-050e-49c6-893b-79ba6fe9b6d2.MP4

EDIT: Upon further inspection via another screen which transitions just a bit slower, it seems the modal screen IS running an exit animation, but it's being moved behind the incoming screen (the screen being navigated back to). The dismissed modal screen is being moved behind everything else. Anyone have a clue what might be causing this?

**Comment by AlexCatch** [comment]
I'm also having this issue, seems to only occur after navigating back from a pushed screen though.
